src/index.js

```diff
@@ -79,14 +80,21 @@ export function startTagRelease( options, queries = questions.general ) {
 }
 
 export function bootstrap() {
-	utils.getGitConfigs()
-		.then( ( [ username, token ] ) => startTagRelease( { username, token } ) )
+	const options = {};
+
+	return pipeline( [
+		::utils.showGitLogs,
+		::utils.getGitConfigs,
+		startTagRelease
+	], options )
 		.catch( error => {
 			utils.prompt( questions.github ).then( answers => {
 				const { username, password } = answers;
 				utils.createGitHubAuthToken( username, password ).then( token => {
 					utils.setGitConfigs( username, token );
-					startTagRelease( { username, token } );
+					options.username = username;
+					options.token = token;
+					startTagRelease( options );
```

Comment:
Yes, from the pipeline above we're able to read the git log even if not authed in github, and those logs still carry over and are stored in `options.log`, and need to be included in the function call `startTagRelease( options )` along with username and token.